MediaPlayer/MediaControls.js

```diff
@@ -31,6 +31,14 @@ const MediaControls = kind({
 	name: 'MediaControls',
 
 	propTypes: /** @lends agate/MediaPlayer.MediaControls.prototype */ {
+		/**
+		 * Removes interactive capability from buttons inside this component.
```

Comment:
Hm, This discription was written in a slightly different from the other sentences. (" Removes interactive capability ")
I hope it is a sentence similar to the existing ones.

Let's just write "disables"

 * Disables MediaControl buttons and becomes non-interactive.
or
*  Disables the media buttons.

 Or please write it in your way